src/AutoUI/models/helpers.ts

```diff
@@ -204,15 +249,54 @@ export const generateSchemaFromRefScheme = (
 	};
 };
 
+export const getRefSchema = (schema: JSONSchema, refSchemePrefix: string) => {
+	const refScheme = parseDescriptionProperty(schema, 'x-ref-scheme') as
+		| string
+		| null;
+	return refScheme
+		? (get(schema, `${refSchemePrefix}${refScheme}`) as JSONSchema) ?? schema
+		: schema;
```

Comment:
Would this work?
```suggestion
	return (get(schema, `${refSchemePrefix}${refScheme}`) as JSONSchema | null) ?? schema;
```

(added the `| null` to the cast too because it has to be able to be nullish for `??` to trigger)

Reply:
> Would this work?

Because if `!refScheme` then `${refSchemePrefix}${refScheme}` = `${refSchemePrefix}undefined` which shouldn't exist right?

Reply:
another very wrong left over, thanks for pointing it out